src/app-layer.c

```diff
@@ -986,6 +988,7 @@ int AppLayerHandleUdp(ThreadVars *tv, AppLayerThreadCtx *tctx, Packet *p, Flow *
     if (r < 0) {
         ExceptionPolicyApply(p, g_applayerparser_error_policy, PKT_DROP_REASON_APPLAYER_ERROR);
         AppLayerIncrErrorExcPolicyCounter(tv, f, g_applayerparser_error_policy);
+        AppLayerIncFlowCounter(tv, f);
```

Comment:
This seems to count too many flows :

I have a flow with 3 packets
- first one is DNS ok, the flow gets counted (// If the other side is unknown, this is the first packet of the flow)
- an error packet: the flow gets counted again
- another error packet, the flow gets counted again...

Reply:
hmm I thought in case of multiple error pkts like that, the protocol parser would be disabled but this is UDP. Did not account for this indeed. Thank you!

Reply:
> > hmm I thought in case of multiple error pkts like that, the protocol parser would be disabled but this is UDP
> 
> Gets disabled on first TCP error, but not for UDP... right ?

right!